tests/data/general-config-only/nomenclature.yaml

```diff
@@ -4,6 +4,5 @@ repositories:
 definitions:
   region:
     repository: common-definitions
-    country: true
```

Comment:
As an alternative you could leave this in and use the pin feature to pin this config to the latest pre-G20 common-definitions revision:
```yaml
repositories:
  common-definitions:
    url: https://github.com/IAMconsortium/common-definitions.git/
    hash: 4f77be8
definitions:
  region:
    repository: common-definitions
    country: true
  variable:
    repository: common-definitions
```

Reply:
We do have a separate test for `country: true`, so I'd rather keep the tests consistent with the latest version of common-definitions to be warned about future conflicts early. There isn't so much extra benefit to the joint test of common-definitions and `country: true`.